### PR TITLE
Add input feature

### PR DESCRIPTION
**Pull request by palomacione**

### CODE REVIEW

**Comment by palomacione** [comment]
[[Docs] Create Input documentation](https://trello.com/c/wFrzF1bd/639-docs-create-input-documentation)

File: docs/beta_input_endpoints.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+## About
+All HTTP methods should be prepended by this service's endpoint:
+
+```
+https://api.withleaf.io/services/beta/api/input
+```
+
+See below the REST resources and their endpoints available in this service.
+
```
Comment (fabyan-alexander) [inline docs/beta_input_endpoints.md:23]: Considering that we will use only /api/users/{leafUserId}... we don't need this group, correct?

**Review by fabyan-alexander — APPROVED** [review]
Great and clean work!

File: docs/beta_input_endpoints.md (hunk 1 above)
Comment (palomacione) [reply docs/beta_input_endpoints.md:23]: true that

File: docs/beta_input_endpoints.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+[1]: https://github.com/Leaf-Agriculture/Leaf-quickstart-Postman-collection
+[2]: #get-all-varieties
+[3]: #get-all-products
+
+
+## About
+All HTTP methods should be prepended by this service's endpoint:
+
+```
+https://api.withleaf.io/services/beta/api/input
```
Comment (Vinniekun) [inline docs/beta_input_endpoints.md:19]:
```suggestion
https://api.withleaf.io/services/beta/api
```

File: docs/beta_input_endpoints.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    "id": "UUID",
+    "leafUserId": "UUID",
+    "name": "str",
+    "crops": "[str, str]"
+  }
+]
+```
+
+### Get All Products
+&nbsp<span class="badge badge--success">GET</span> `/users/{leafUserId}/products`
```
Comment (Vinniekun) [inline docs/beta_input_endpoints.md:128]:
```suggestion
<span class="badge badge--success">GET</span> `/users/{leafUserId}/products`
```
It is displaying like this:
image

File: docs/beta_input_endpoints.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+Description | Endpoints
+--- | ---
+[Get all Varieties][1] | <span class="badge badge--success">GET</span> `/users/{leafUserId}/varieties`
+[Get all Products][2] | <span class="badge badge--success">GET</span> `/users/{leafUserId}/products`
+
+
+### Get All Varieties
+
+&nbsp<span class="badge badge--success">GET</span> `/users/{leafUserId}/varieties`
```
Comment (Vinniekun) [inline docs/beta_input_endpoints.md:38]:
```suggestion
<span class="badge badge--success">GET</span> `/users/{leafUserId}/varieties`
```
It is displayed like this:
image

**Review by Vinniekun — APPROVED** [review]

File: docs/beta_input_endpoints.md (hunk 2 above)
Comment (palomacione) [reply docs/beta_input_endpoints.md]: https://github.com/Leaf-Agriculture/docs/pull/219/commits/4fec0484efebfaf552ffc5fe5d1a6e78ff790294

**Review by gustavorodriguespereira — APPROVED** [review]

**Pull request merged**